### DISCUSSION TITLE
How can I remove added word from spellchecker's dictionary?

### DISCUSSION THREAD

**Discussion by dogfuntom** [TriliumNext/Trilium#3039] [Q&A] [answered]
I accidentally added a wrong spelling for a word. How can I undo adding it?

P.S.: I looked into Data Directory and into the SQLite DB but didn't find anything that looks like a list of custom words to me.

**Comment by sigaloid** [comment, marked as answer]
If you're on windows, previous issues suggest the location is AppData\Roaming\Trilium Notes\Custom Dictionary.txt

**Comment by dogfuntom** [comment]
Thanks!
Didn't know there are two directories in Roaming instead of the usual one directory per app (or even a single folder per publisher).
